### PR TITLE
fix: config acl

### PR DESCRIPTION
**Pull request by jiefenghuang**

### CODE REVIEW

File: cmd/config.go [hunk 1]
```diff
@@ -243,10 +243,15 @@ func config(ctx *cli.Context) error {
...
-				format.EnableACL = true
-				format.MinClientVersion = "1.2.0-A"
+			if enableACL := ctx.Bool(flag); enableACL != format.EnableACL {
+				if enableACL {
+					msg.WriteString(fmt.Sprintf("%s: %v -> %v\n", flag, format.EnableACL, true))
+					msg.WriteString(fmt.Sprintf("%s: %s -> %v\n", "min-client-version", format.MinClientVersion, "1.2.0-A"))
+					format.EnableACL = true
+					format.MinClientVersion = "1.2.0-A"
+				} else {
+					fmt.Println("cannot disable acl")
```
Comment (SandyXSD) [inline cmd/config.go:253]: better return an error.

File: cmd/config.go [hunk 2]
```diff
@@ -243,10 +243,15 @@ func config(ctx *cli.Context) error {
...
 			}
 		case "enable-acl":
-			// cannot disable
-			if ctx.Bool(flag) {
-				format.EnableACL = true
-				format.MinClientVersion = "1.2.0-A"
+			if enableACL := ctx.Bool(flag); enableACL != format.EnableACL {
+				if enableACL {
+					msg.WriteString(fmt.Sprintf("%s: %v -> %v\n", flag, format.EnableACL, true))
+					msg.WriteString(fmt.Sprintf("%s: %s -> %v\n", "min-client-version", format.MinClientVersion, "1.2.0-A"))
```
Comment (SandyXSD) [inline cmd/config.go:249]:
```suggestion
					msg.WriteString(fmt.Sprintf("%s: %s -> %s\n", "min-client-version", format.MinClientVersion, "1.2.0-A"))
```

Comment (jiefenghuang) [reply cmd/config.go]: done

File: cmd/config.go (hunk 1 above)
Comment (jiefenghuang) [reply cmd/config.go:253]: i think this is enough, print msg and nothing changed.

image

**Review by zhoucheng361 — APPROVED** [review]

**Pull request merged**